frontend/src/components/LandingPage.css

```
@@ -189,394 +171,168 @@
-  flex: 1;
+/* ── Bento Cards (Ultimate Glassmorphism) ── */
+.lp-card {
+  background: rgba(255, 255, 255, 0.6);
+  backdrop-filter: saturate(180%) blur(40px);
+  -webkit-backdrop-filter: saturate(180%) blur(40px);
+  border: 1px solid rgba(255, 255, 255, 0.9);
+  border-radius: 36px; /* High radius like iOS widgets */
+  padding: 36px;
+  box-shadow: 0 20px 40px rgba(0, 0, 0, 0.04), 
+              inset 0 1px 0 rgba(255, 255, 255, 1); /* Outer shadow + Inner highlight */
   display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)